source/common/protobuf/utility.h

```diff
@@ -68,6 +68,18 @@ class MessageUtil {
   static void loadFromFile(const std::string& path, Protobuf::Message& message);
 
   /**
+   * Convert from google.protobuf.Any to a typed message.
+   * @param message source google.protobuf.Any message.
+   * @return MessageType the typed message inside the Any.
+   */
+  template <class MessageType>
+  static inline MessageType anyConvert(const ProtobufWkt::Any& message) {
+    MessageType typed_message;
+    message.UnpackTo(&typed_message);
```

Comment:
Q: I assume this needs some type of error checking?

Reply:
Added.